### DISCUSSION TITLE
Avoiding the Command Prompt while tufup overwriting the installed new file inside Temp folder in place of original file.

### DISCUSSION THREAD

**Discussion by ritishbhardwaj** [dennisvang/tufup#137] [Q&A] [answered]
While overwriting the new file in place of old file, a Command Prompt window [For reference, kindly see attached pic] automatically pops up. I do not want it to show as i want seamless installation. Please help me how can i fix this issue, like in which file should i make changes (of framework or the client code written by me).

![Screenshot (34)](https://github.com/dennisvang/tufup/assets/81488252/832b3fac-90d1-4c65-aba0-ef759c21448b)
[https://drive.google.com/file/d/1rWtsp57E0xpcNKwSA4fP6wBFs9fjFg1x/view?usp=sharing](url)      mentioning the same screenshot if in case any issue arise here

**Comment by dennisvang** [comment, marked as answer]
@ritishbhardwaj Sorry for the late reply, I was away for some time.

To prevent the command window from showing, have a look at the description for #90 

Also see in https://github.com/dennisvang/tufup/discussions/96#discussioncomment-7883232

**Reply by ritishbhardwaj** [reply]
Thanks for this......
It works fine for me

There is one cons that i have found here, Suppose for some reason the file which is downloaded have not replaced the old .exe to new .exe due to reasons like the process is already being run [as discussed in #140 ], in this case we won't get what error is coming up as no console is getting opened [to see error check #140 ] while updation is in process. 

But still this solution works for me.
Thanks.
